api.ts

```diff
@@ -1619,6 +1630,8 @@ export interface DynamicSyncTableOptions<
    * in placeholderSchema will be rendered by default after the sync.
    */
   placeholderSchema?: SchemaT;
+
+  autocomplete?: (context: CellAutocompleteExecutionContext, search: string) => Promise<any[]>;
```

Comment:
Is this supposed to be `CellAutocompleteMetadataFunction`?

Reply:
Yes, that'll be fixed in the next version. I'm also removing "search" as a parameter here because I want to make it a property on the context so I can track if a pack reads it or not for caching dependencies

Reply:
Done